op-node/node/node.go

```diff
@@ -378,6 +378,10 @@ func (n *OpNode) RequestL2Range(ctx context.Context, start, end eth.L2BlockRef)
 		return n.rpcSync.RequestL2Range(ctx, start, end)
 	}
 	if n.p2pNode != nil && n.p2pNode.AltSyncEnabled() {
+		if time.Unix(int64(start.Time), 0).Before(time.Now().Sub(14 * time.Hour)) {
```

Comment:
TODO: Decide on the value here. 12hrs is goerli/mainnet sequencing window. I gave it a buffer on top of that. We could also go down to like 6 hrs.